partials/_create.scss

```diff
@@ -0,0 +1,40 @@
+.create-list {
+    width: 100%;
+    text-align: center;
+    font-family: $font-regular;
+    font-size: 14px;
+    line-height: 24px;
+
+    &__header {
+        margin: 24px 0px 32px 0px;
+    }
+
+    &__chat {
+        padding: 24px 0px 96px;
+        width: 347px;
+        margin: auto;
+    }
+
+    &-btn {
+        width: 142px;
+        margin: auto;
+    }
+}
+
+.underline {
+    text-decoration: underline;
+    cursor: pointer;
+
+    &:hover {
+        color: $tws_coral;
+    }
+}
+
+.bow-large {
+    height: 30px;
+    width: 58px;
+}
+
+.create--margin {
```

Comment:
1) I think the classname is misleading, if anything it should be called 'noMargin'
2) is there any block with a class .create? Or is this an independent class? If the first is true, you should write it like
```
.create {
  ...some style 
  
  &--margin {
    ...
    }
}
```
if the latter is true, you should change the classname - currently it looks like a bem modifier

Reply:
I'm actually not sure why I've done this, can't find this classname in my html...

Reply:
you can go ahead and remove it then :)